### DISCUSSION TITLE
Trail without fillColor

### DISCUSSION THREAD

**Discussion by Celine-Lucas** [matteobruni/tsparticles#2446] [Q&A] [answered]
Hi, 
Is there a way to make trail option fillColor to transparent or radial-gradient background ? 
I have a radial-gradient background and when i enable trail, it makes the background all same color (the one specified in fillColor option)

**Comment by matteobruni** [comment, marked as answer]
Hi @Celine-Lucas, it’s not possible yet, but I was discussing this on Discord this morning.

**Reply by Celine-Lucas** [reply]
Ok thanks for this fast answer !
